lib/libutee/include/scp03_pta.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (C) 2020, Foundries Limited
+ */
+
+#ifndef __SCP03_PTA_CLIENT_H
+#define __SCP03_PTA_CLIENT_H
```

Comment:
file name is **scp03_pta.h**, should it be `__SCP03_PTA_H` here?

Reply:
sure, will update